kibana/Dockerfile

```diff
@@ -1,7 +1,5 @@
 FROM kibana:5
 
-RUN apt-get update && apt-get install -y netcat bzip2
-
 COPY entrypoint.sh /tmp/entrypoint.sh
```

Comment:
You can also remove the lines 3-4 which are used to copy the `entrypoint.sh` script. This script has no purposes anymore, so please delete it too.